types/wegame-api/tslint.json

```diff
@@ -1,3 +1,6 @@
 {
-    "extends": "dtslint/dt.json"
+    "extends": "dtslint/dt.json",
+    "rules": {
+    	"dt-header": false
```

Comment:
Please don't disable any rules. I suggest instead just using `2.6`, without the patch.

Reply:
thanks for reviewing on this PR @armanio123 , actually I found chai-jquery uses this way to modify the dt-header: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/34112, so I did this to keep the version number newest.
this definition is for a middleware API of a software, no js lib, and the version number changes very often, like 1-2 week a release 2.6.6 2.6.7 2.6.8, so I have to change my lib that often, so I think it's better to keep the minor versoin number to avoid confuse.

Reply:
Disabling dt-header will just hide the problem, unfortunately. Definitely Typed uses the patch version itself [1], so it's impossible to have types that target only a patch version. They must work for all patches of a minor version. 

Does wegame-api really add or break things in patch versions? You might have to just add things from newer patch versions and expect that users who are on older patch versions will not be warned if they incorrectly use these new properties.

[1] Definitely Typed will ignore the patch version you provide and, starting from 0, increment the patch version for every update of the same minor version.

Reply:
@sandersn oh! so actually the version in .d.ts is the version for the definition but not the JS Lib version?

Reply:
That's correct the version on the d.ts. is the version for the definition.

Reply:
I got it, I'm sorry for the confusion, will do the changes